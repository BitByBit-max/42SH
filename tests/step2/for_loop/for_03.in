for i in 1 2 3 4 5; 
do 
	if echo "who?"; then
		echo hi;
	else
		echo bye
	fi
done
