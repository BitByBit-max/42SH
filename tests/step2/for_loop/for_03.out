who?
hi
who?
hi
who?
hi
who?
hi
who?
hi
